packages/ui/src/utils/querystring.ts

```diff
@@ -0,0 +1,18 @@
+export interface Query {
+  [_: string]: number | undefined | string | string[];
+}
+
+export function formatQuerystring(search: Query): string {
```

Comment:
Recommend to use a library, or add test code for this function (in real life project).

Reply:
Agree